audits/judging/judging/README.md

```diff
@@ -18,49 +18,57 @@ This guide aims to provide clarity for both Watsons & protocols on various categ
 
-***
+### III. Sherlock's standards:
 
-### III. Some standards observed:
-
-1. **Hierarchy of truth:** Contest README > Sherlock rules for valid issues > Historical decisions. \
+1. **Hierarchy of truth:** Contest README > Sherlock rules for valid issues. \
    While considering the validity of an issue in case of any conflict the sources of truth are prioritized in the above order. \
-   For example: In case of conflict between Sherlock rules vs Sherlock's historical decision,  Sherlock criteria for issues must be considered the source of truth. \
-   In case of conflict between information in the README vs Sherlock rules, the README overrides Sherlock rules. \
-   Also, in case of any updates in the rule book:
-   1. If the updated rules are in conflict with historical decisions then the new rules apply only to contests that start after the date of change. \
+   For example: In case of conflict between information in the README vs Sherlock rules, the README overrides Sherlock rules. \
+   **Exception**: Sometimes the README would take a wider group of impact/issue types out of scope than intended. In those cases, Sherlock may decide to consider an issue valid, while it would otherwise be considered out of scope. [Example(Valid)](https://github.com/sherlock-audit/2023-10-looksrare-judging/issues/136) \
+   1. If rules are updated, the new rules apply only to contests that start after the date of change. \
       Please check [criteria-changelog.md](criteria-changelog.md "mention") for information on the latest changes in the judging criteria/rules.
-2. **Could Denial-of-Service (DOS), griefing, or locking of contracts count as a Medium (or High) issue?** It would not count if the DOS, etc. lasts a known, finite amount of time <1 year. If it will result in funds being inaccessible for >=1 year, then it would count as a loss of funds and be eligible for a Medium or High designation. The greater the cost of the attack for an attacker, the less severe the issue becomes.
+   
+   **Historical decisions are not considered sources of truth.**
+2. **Could Denial-of-Service (DOS), griefing, or locking of contracts count as a Medium (or High) issue?** DoS has two separate scores on which it can become an issue:
+   1. The issue causes locking of funds for users for more than a week.
+   2. The issue impacts the availability of time-sensitive functions (cutoff functions are not considered time-sensitive).
+If at least one of these are describing the case, the issue can be a Medium. If both apply, the issue can be considered of High severity. Additional constraints related to the issue may decrease its severity accordingly. \
+Griefing for gas (frontrunning a transaction to fail, even if can be done perpetually) is considered a DoS of a single block, hence only if the function is clearly time-sensitive, it can be a Medium severity issue.
```

Comment:
> Griefing for gas (frontrunning a transaction to fail, even if can be done perpetually) is ...
I think this should just be: 
"Griefing Attacks (frontrunning a transaction to fail, even if can be done perpetually) are ...

Frontrunning a TX to make it fail is not necessarily related to Gas Griefing.

Reply:
Can you elaborate?

Reply:
> Can you elaborate?

What I mean by that is, that gas griefing is, if the attacker is doing something (like frontrunning tx to fail) to make the victim pay more/useless gas.

In a DoS Scenario an attacker could frontrun a TX to make the victims tx fail for other reasons, than just to grief gas, e.g preventing a time critical action to be done.

Reply:
It's already taken into account in the rules update:
> only if the function is clearly time-sensitive, it can be a Medium severity issue

Reply:
> It's already taken into account in the rules update:
> 
> > only if the function is clearly time-sensitive, it can be a Medium severity issue

Yeah, that is good!
I just wondered, cause the mentioning of Gas Griefing together with the DoS and making transactions fail seemed a bit off for me.